app/views/about/index.html.haml

```diff
@@ -14,15 +14,24 @@
         %div
           = image_tag 'hackweek-label.png', class: 'img-responsive', alt: 'Hack Week'
 .row
-  .col-md-12
+  .col-md-7
     %p.lead
       Hack Week is the time SUSE engineers experiment, innovate &amp; learn interruption-free
       for a whole week! Across teams or alone, but always without limits.
     %p.lead
       A SUSE tradition since 2007, Hack Week has brought together hundreds of hackers around
       the globe to collaborate on open source.
+    %p.lead
+      Hackweek 17 is coming up! It starts Monday, July 9th until Friday, July 13th.
     %p.lead
       What are you going to hack on?
+    %p.lead
```

Comment:
I would move this up to the previous paragraph. This is a call to action. No need to bring up something else in between.

Reply:
I am fine with that, but can not do it any more soon. I am in the US next week. Could you do the changes you wanna see and deploy it? Thanks!